apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -35,9 +205,217 @@ export default class ProjectCard extends React.Component {
+                        checked={checkbox.checked}
+                        onChange={() =>
+                          this.handleCheckboxChange(checkbox.name)
+                        }
+                        style={{
+                          marginRight: 10,
+                          marginTop: 0,
+                          accentColor: '#0093A4',
+                        }}
+                      />
+                      <span
+                        style={{
+                          flex: '1',
+                          ...styles.popUpBody,
+                          verticalAlign: 'middle',
+                        }}
+                      >
+                        {checkbox.name.replace(/-/g, ' ')}
+                      </span>
+                    </label>
+                  ))}
+                </div>
+                {isAnyCheckboxSelected ? (
+                  <div
+                    style={{padding: 5}}
+                    className="g-recaptcha"
+                    data-sitekey={captchaSiteKey}
+                    data-callback="onCaptchaSubmit"
+                    data-expired-callback="onCaptchaExpired"
+                  />
+                ) : null}
+                <hr style={styles.popUpLines} />
+                <div style={{display: 'flex', justifyContent: 'flex-end'}}>
+                  <Button
+                    onClick={this.cancel}
+                    text={'Cancel'}
+                    color={Button.ButtonColor.white}
+                    style={{
+                      borderColor: color.neutral_dark,
+                      color: color.neutral_dark,
+                    }}
+                  />
+                  <Button
+                    onClick={this.handleSubmit}
+                    disabled={!submitButtonEnabled}
+                    text={'Submit'}
+                    style={{outline: 'none'}}
+                    color={Button.ButtonColor.brandSecondaryDefault}
+                  />
+                </div>
+              </div>
+            )}
+          </AccessibleDialog>
+        ) : null}
+
         <div style={styles.card}>
+          {!showSubmittedHeader ? (
+            <div
+              style={{
+                ...thumbnailStyle,
+                height: 40,
+                display: 'flex',
+                alignItems: 'center',
+                justifyContent: 'flex-end',
+              }}
+            >
+              <button
+                type="button"
+                onClick={this.showReportAbusePopUp}
+                style={{
+                  ...styles.transparentButton,
+                  padding: 6,
+                  marginRight: 16,
+                  boxShadow: 'none',
+                  background: reportButtonPressed
+                    ? '#fbe0dd'
+                    : reportButtonHovered
+                    ? '#fbe0dd'
+                    : 'none',
+                }}
+                onMouseEnter={this.handleButtonMouseEnter}
```

Comment:
Once you refactor this button to be styled with scss modules instead of inline styles - you won't need all these callbacks and component state, css handle hover/active(pressed)/focused states pretty good out of the box